ios/NebulaNetworkExtension/PacketTunnelProvider.swift

```diff
@@ -13,15 +13,12 @@ class PacketTunnelProvider: NEPacketTunnelProvider {
     private var didSleep = false
     private var cachedRouteDescription: String?
     
-    // This is the system completionHandler, only set when we expect the UI to ask us to actually start so that errors can flow back to the UI
-    private var startCompleter: ((Error?) -> Void)?
-    
-    override func startTunnel(options: [String : NSObject]?, completionHandler: @escaping (Error?) -> Void) {       
+    override func startTunnel(options: [String : NSObject]?, completionHandler: @escaping (Error?) -> Void) {
         // There is currently no way to get initialization errors back to the UI via completionHandler here
         // `expectStart` is sent only via the UI which means we should wait for the real start command which has another completion handler the UI can intercept
-        // In the end we need to call this completionHandler to inform the system of our state
         if options?["expectStart"] != nil {
```

Comment:
Is there any advantage at all to keeping these two code paths right now? Or should we just remove the concept of expectStart altogether for now?